fetcher/src/job/job.api.ts

```diff
@@ -98,18 +98,23 @@ export async function deactivateAggregator(aggregatorHash: string, chain: string
   return await updateAggregator(aggregatorHash, chain, false)
 }
 
-export async function fetchDataFeed({
+export async function loadLastSubmissionData({
   aggregatorHash,
   logger
 }: {
   aggregatorHash: string
   logger: Logger
 }): Promise<IAggregate> {
   try {
-    const url = buildUrl(process.env.ORAKL_NETWORK_API_URL, `aggregate/${aggregatorHash}/latest`)
```

Comment:
I think we do not use this endpoint anywhere, so how about we remove it since we would not need it anymore after this update?